### PR TITLE
Fix incorrect UI for Automation Track on Mac OS X

### LINKED ISSUE #2503: Documentation/UI incorrect for Automation Track on Mac OS X
The documentation and UI say to "`Drag a control while pressing <Ctrl>`" onto an Automation Track.

On Mac OS X the correct key to press appears to be the "Command" ("Butterfly") key. Although this may depend on whether the "control" key has been configured to act as a second mouse button.

There may be other places where this is an issue but this is where I got confused for a while. :)

Using 1.1.3 but the text appears to be the same in trunk.

Thanks.

### PR DESCRIPTION
**Pull request by Umcaruje**
Resolves #2503. 

Nothing too fancy, just followed @tresf's instructions. 

Don't own a Mac, so can't test it there, but I tested on linux and it worked fine.

### CODE REVIEW

**Comment by tresf** [comment]
While we're at it... :smiling_imp: 

https://github.com/LMMS/lmms/blob/master/src/gui/widgets/ProjectNotes.cpp#L119
https://github.com/LMMS/lmms/blob/master/src/gui/TimeLineWidget.cpp#L327
https://github.com/LMMS/lmms/blob/master/src/tracks/SampleTrack.cpp#L269

**Comment by Umcaruje** [comment]
K, I'll look into fixing the rest tomorrow.

**Comment by Umcaruje** [comment]
Ok, updated all of the files and translations. I think I found all the spots. Let me know if I missed something.

File: src/gui/editors/PianoRoll.cpp [hunk 1]
```diff
@@ -1870,7 +1870,7 @@ void PianoRoll::mouseReleaseEvent( QMouseEvent * me )
 		{
 			// select the notes within the selection rectangle and
 			// then destroy the selection rectangle
-			computeSelectedNotes(
```
Comment (tresf) [inline src/gui/editors/PianoRoll.cpp]: This line looks the same as well as several others in this file...

File: src/gui/widgets/ProjectNotes.cpp [hunk 2]
```diff
@@ -2,7 +2,7 @@
  * ProjectNotes.cpp - implementation of project-notes-editor
  *
  * Copyright (c) 2005-2008 Tobias Doerffel <tobydox/at/users.sourceforge.net>
- *
```
Comment (tresf) [inline src/gui/widgets/ProjectNotes.cpp]: This line looks the same...

**Comment by tresf** [comment]
A few formatting questions, but it looks good to merge.

File: src/gui/editors/PianoRoll.cpp (hunk 1 above)
Comment (Umcaruje) [inline src/gui/editors/PianoRoll.cpp]: @tresf these lines actually had a trailing space, so atom (my text editor) fixed that.

**Comment by Umcaruje** [comment]
So the random lines that got edited just got their trailing space removed, that's all there is to it.

**Comment by Umcaruje** [comment]
@tresf I removed the formatting changes. Is there anything else, or can this be merged?

**Comment by tresf** [comment]
> @tresf I removed the formatting changes. Is there anything else, or can this be merged?

There are actually a few left in there but I won't nit-pick that.

I did however notice that some of the `<unfinished>` translations have `Ctrl` replaced whereas some do not.  Was there a reason for this, or was it just a matter of search/replace?  Anyhow, I think it's ready to be merged.

**Comment by Umcaruje** [comment]
>I did however notice that some of the <unfinished> translations have Ctrl replaced whereas some do not. 

I did a global search/replace in the data/locale folder for `Ctrl` and replaced it with `%1` Could you comment on the lines where there is still a Ctrl left? I can't seem to find them.

File: data/locale/ca.ts [hunk 3]
```diff
@@ -5322,7 +5322,7 @@ Reason: &quot;%2&quot;</source>
         <translation type="unfinished">Enganxa</translation>
     </message>
     <message>
-        <source>Mute/unmute (&lt;Ctrl&gt; + middle click)</source>
+        <source>Mute/unmute (&lt;%1&gt; + middle click)</source>
```
Comment (tresf) [inline data/locale/ca.ts]: Here's one.

Comment (Umcaruje) [inline data/locale/ca.ts]: Uh, yeah, that's weird, lemme redo it...

**Comment by Umcaruje** [comment]
Ok, it seems that I forgot to do a global search/replace. Did it now. Note: some lines in the arabic translation were deliberately left the same, since they would get screwed up when edited. Also in the german translation the translator used `Strg` as a translation to `Ctrl`, so I think its up to to the translators to fix these issues.

**Comment by midi-pascal** [comment]
@Umcaruje Just my 2 cents here: Are you aware that the "ts" files are generated automatically by Qt Linguist (lupdate extracts the literals from the source files and rewrite the "ts" files)? So, chances are your changes to the translation files will be overridden during the next update...
I hope I am wrong but I use lupdate regularly to keep the translation files up to date :cry:

**Comment by Umcaruje** [comment]
>Just my 2 cents here: Are you aware that the "ts" files are generated automatically by Qt Linguist 

Nope, had no idea.

>lupdate extracts the literals from the source files and rewrite the "ts" files)? So, chances are your changes to the translation files will be overridden during the next update...

Well, I also changed the literals in the source files, so that shouldn't be a problem, as far as I understand. 

If it does overwrite the changes, well, there's nothing I can do about it, and translators will need to sort that out, I guess.

**Comment by midi-pascal** [comment]
@Umcaruje That is Ok if you made the changes to the source files too. As one of the French translators I will check fr.ts after the merge and do adjustments if required.

**Comment by tresf** [comment]
> @Umcaruje Just my 2 cents here: Are you aware that the "ts" files are generated automatically by Qt Linguist (lupdate extracts the literals from the source files and rewrite the "ts" files)?

If that's the case, I believe this to only be true for defaults (English, unfinished), otherwise your logic would have all translation work be pointless.

File: src/core/Track.cpp [hunk 4]
```diff
@@ -933,7 +943,12 @@ void TrackContentObjectView::contextMenuEvent( QContextMenuEvent * cme )
 					tr( "Paste" ), m_tco, SLOT( paste() ) );
 	contextMenu.addSeparator();
 	contextMenu.addAction( embed::getIconPixmap( "muted" ),
-				tr( "Mute/unmute (<Ctrl> + middle click)" ),
+				tr( "Mute/unmute (<Ctrl> + middle click)" ).arg(
```
Comment (tresf) [inline src/core/Track.cpp]: This should be `%1`

**Comment by tresf** [comment]
> I think it's ready to be merged.

I only found one missing one.  Looks good after that.

**Comment by Umcaruje** [comment]
>I only found one missing one.

Fixed it :+1:

**Pull request merged**